pages/charts.tsx

```diff
@@ -119,7 +119,7 @@ function Charts(props: Props) {
 // https://nextjs.org/docs/api-reference/data-fetching/getInitialProps
 
 Charts.getInitialProps = async (): Promise<Props> => {
-  const projectReq = await fetch(`http://delta-core.dsch.dev/api/v1/projects`, {
+  const projectReq = await fetch(`${process.env.deltaCore}/api/v1/projects`, {
```

Comment:
Don't tell me that this was me D: hoooo no
I would have to be more careful next time, a thousand apologies for this 😿 
and a million thanks for fixing it!

Reply:
of course it was me, pfff 🙈 
We say, better later than never :) Didn't notice i did it

Reply:
hahahaha I really believed it was me, looks like something I would do. I fixed the test that was failing, so this is ready to go 🚀 
Thanks for adding the current tag on historical tests btw

Reply:
it didn't like your fix either

Reply:
let's have a look tomorrow why it's not clickable

Reply:
Yep, its picky, I'm adding wait everywhere now 🔫

Reply:
🔧

Reply:
your prayers worked! thanks, and good night